docs/proposals/nginx-extensions.md

```diff
@@ -0,0 +1,964 @@
+| Client keepalives                                                                                |                     |
+| TLS settings. For example, TLS protocols and server ciphers                                      |                     |
+| OpenTelemetry tracing                                                                            |                     |
+
+### Medium-Priority Features
+
+| Features                                                                               | Requires NGINX Plus |
+|----------------------------------------------------------------------------------------|---------------------|
+| Circuit breaker                                                                        |                     |
+| Load-balancing method                                                                  |                     |
+| Load-balancing method (least time)                                                     | X                   |
+| Limit connections to a server                                                          |                     |
+| Request queuing. Queue requests if servers are unavailable                             | X                   |
+| Basic authentication. User/Password                                                    |                     |
+| PROXY protocol                                                                         |                     |
+| Upstream zone size                                                                     |                     |
+| Custom response. Return a custom response for a given path                             |                     |
+| External routing. Route to services outside the cluster                                |                     |
+| HTTP Strict Transport Security (HSTS)                                                  |                     |
+| API Key authentication                                                                 |                     |
+| Next upstream retries. NGINX retries by trying the _next_ server in the upstream group |                     |
+| Proxy Buffering                                                                        |                     |
+| Connection timeouts                                                                    |                     |
+
+### Low-Priority Features
+
+| Features                                   | Requires NGINX Plus |
+|--------------------------------------------|---------------------|
+| Pass/hide headers to/from the client       |                     |
+| Custom error pages                         |                     |
+| Access control                             |                     |
+| Windows Technology LAN Manager (NTLM)      | X                   |
+| Backend client certificate verification    |                     |
+| NGINX worker settings                      |                     |
```

Comment:
not sure where this directive is reflected https://github.com/nginxinc/nginx-gateway-fabric/blob/de23bce0409b91cefa4e01bc5785f07d55eac4ee/internal/mode/static/nginx/conf/nginx.conf#L9 , but by default, this means one worker can only have 1024 connections, including both from clients and to backends, which is limiting in case of high enough RPS :(

Reply:
I had it lumped in with worker settings. Do you think it deserves its own entry in this table?

Reply:
sounds good